TCAT.xcodeproj/project.pbxproj

```diff
@@ -996,7 +996,7 @@
 				INFOPLIST_FILE = TCAT/Supporting/Info.plist;
 				INFOPLIST_KEY_CFBundleDisplayName = "Ithaca Transit";
 				INFOPLIST_KEY_LSApplicationCategoryType = "public.app-category.travel";
-				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 18.0;
```

Comment:
Is there a reason why we modified the deployment target? We want to make sure this isn't too new since may people have not yet upgraded to iOS 18 quite yet

Reply:
Omg I'm not sure why it's like that.. I think my Xcode might've automatically changed it when I updated it -- gonna change that back ASAP!